reef-runtime-mesos/src/main/proto/mesos_runtime.proto

```diff
@@ -0,0 +1,32 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+option java_package = "org.apache.reef.runtime.mesos.proto";
+option java_outer_classname = "ReefRuntimeMesosProtocol";
+option java_generic_services = true;
+option java_generate_equals_and_hash = true;
+
+message EvaluatorLaunchProto {
```

Comment:
We are trying to reduce our dependency on protobuf, as it is a bit cumbersome in terms of onboarding new developers (they need `protoc`). Is it feasible to replace this with an Avro schema? Avro has a pure-java Maven plugin that makes life a lot easier for us.